src/shared/components/heroBanner/heroBanner.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './heroBanner.css';
+
+const propTypes = {
```

Comment:
for the sake of intellisense in editors, I'd prefer these variables be named with a `_` in front to discern between the `prop-types` API and the actual variable names.

Reply:
Can you explain a little more? When I add an underscore before `propTypes` ES Lint throws an error, "Unexpected dangling '_' in '_propTypes' no-underscore-dangle"

Reply:
Looks like we might want to turn off this ESLint rule: https://eslint.org/docs/rules/no-underscore-dangle